utils/docker/images/prepare-pmemkv.sh

```diff
@@ -38,10 +38,7 @@ set -e
 
 package_type=$1
 
-# stable_pmemkv_version="0.8"
-
-# commit: Merge pull request #436 from ldorau/Replace-PMEMKV_STATUS_FAILED-with-PMEMKV_STATUS_UNKNOWN_ERROR, 20.09.2019
-current_pmemkv_version="f14ea1c0243f5b968b835d3581a8a066edfc9373"
+stable_pmemkv_version="1.0"
```

Comment:
I know these are comments, but please don't remove them, so we still remember to update it